### PR TITLE
image-set: FIXES get image-sets/{ID} to return not found when imageSet does not exist.

### PR DESCRIPTION
**Pull request by ldjebran**
Fixes # THEEDGE-1096

## Type of change

What is it?

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update
- [ ] Refactor

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] I run `go fmt ./...` to check that my code is properly formatted
- [X] I run `go vet ./...` to check that my code is free of common Go style mistakes

### CODE REVIEW

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
7db747fd-f9dc-4318-8212-6cd6461b2c64
d82ab309-d2c2-4705-9f3d-d9227de3ee82

File: pkg/routes/imagesets.go [hunk 1]
```diff
@@ -83,55 +83,39 @@ var imageStatusFilters = common.ComposeFilters(
...
 			}
-			result := db.DB.Where("account = ? and Image_sets.id = ?", account, imageSetID).Find(&imageSet)
-
-			if result.Error != nil {
-				err := errors.NewNotFound(result.Error.Error())
-				w.WriteHeader(err.GetStatus())
-				if err := json.NewEncoder(w).Encode(&err); err != nil {
-					s.Log.WithField("error", err.Error()).Error("Error while trying to encode")
+			if result := db.DB.Where("account = ? and id = ?", account, imageSetID).
+				Preload("Images").Preload("Images.Installer").
```
Comment (acosferreira) [inline pkg/routes/imagesets.go:105]: do we need to preload all this images info?

Comment (acosferreira) [reply pkg/routes/imagesets.go:105]: I think at imageset level we just need the last installer to get the iso url download. :thinking:

Comment (ldjebran) [reply pkg/routes/imagesets.go:105]: ah OK , will update that , thanks @acosferreira I was not getting the purpose of that, may be not like this

Comment (ldjebran) [reply pkg/routes/imagesets.go]: reverted back  my changes , set only the function that need changes , will update after GA.

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
b730e3fc-87ae-4780-9fc5-72b52b84d82c
bccfe7a4-e5fb-43cb-a4e9-45e87e4a3fd3

**Review by acosferreira — APPROVED** [review]
lgtm

**Comment by InsightsDroid** [comment]
Test results are available in [Ibutsu](https://url.corp.redhat.com/ibutsu-runs). The test run IDs are:
13060b1b-16bd-4139-8d33-091f10586b46
df8c01b4-83d6-46c6-88c2-e65f697b4925

**Pull request merged**